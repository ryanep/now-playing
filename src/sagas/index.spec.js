import rootSaga from './index';

describe('sagas/index', () => {
  it('exposes the expected interface', () => {
    expect(rootSaga).toBeInstanceOf(Function);
  });

  it('exposes the expected shape', () => {
    const gen = rootSaga();
    expect(gen.next().value).toMatchSnapshot();
  });
});
