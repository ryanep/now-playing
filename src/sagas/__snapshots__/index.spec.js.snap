// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sagas/index exposes the expected shape 1`] = `
Object {
  "@@redux-saga/IO": true,
  "ALL": Array [
    Object {
      "@@redux-saga/IO": true,
      "FORK": Object {
        "args": Array [
          "TRACK_REQUEST",
          [Function],
        ],
        "context": null,
        "fn": [Function],
      },
    },
    Object {
      "@@redux-saga/IO": true,
      "FORK": Object {
        "args": Array [
          "ACCESS_TOKEN_REQUEST",
          [Function],
        ],
        "context": null,
        "fn": [Function],
      },
    },
    Object {
      "@@redux-saga/IO": true,
      "FORK": Object {
        "args": Array [
          "ACCESS_TOKEN_EXPIRED",
          [Function],
        ],
        "context": null,
        "fn": [Function],
      },
    },
  ],
}
`;
